corehq/apps/integration/templates/payments/payments_verify_report.html

```diff
@@ -5,6 +5,15 @@
 {% js_entry "integration/js/payments/payments_verify" %}
 
 {% block page_content %}
+  {% if not has_config %}
+  <p class="alert alert-warning">
+    {% blocktrans %}
+    To request payments successfully, link this feature to a Connection Setting in your configuration by going to the
```

Comment:
just suggesting a minor rephrase

`To request payments successfully, add necessary settings on the <a href="{{ config_url }}">Payments Configuration</a> page`

Reply:
https://github.com/dimagi/commcare-hq/pull/35984/commits/d79e362af52d43b44b438cc6278c33842a518fad